### PR TITLE
Add proper activation function

### PR DESCRIPTION
**Pull request by kyscg**
In this load_data/csv tutorial (https://www.tensorflow.org/tutorials/load_data/csv), the final prediction values are weird. For instance:
```
Predicted survival: -236.86% | Actual outcome: SURVIVED
Predicted survival: -100.17% | Actual outcome: DIED
Predicted survival: -238.74% | Actual outcome: DIED
Predicted survival: 147.69% | Actual outcome: DIED
Predicted survival: 12.94% | Actual outcome: SURVIVED
```

This PR adds the `sigmoid` activation function to the last `Dense` layer, keeping the result between 0 and 1:

```
Predicted survival: 0.01%  | Actual outcome:  SURVIVED
Predicted survival: 100.00%  | Actual outcome:  DIED
Predicted survival: 0.02%  | Actual outcome:  SURVIVED
Predicted survival: 99.86%  | Actual outcome:  DIED
Predicted survival: 2.43%  | Actual outcome:  SURVIVED
```

### CODE REVIEW

**Comment by lamberta** [comment]
We're intentionally using logits—though I agree the final section reads weird. I think it would be better to convert it to something more interpretable when displaying the results and not in the model. @MarkDaoust wdyt?

**Comment by MarkDaoust** [comment]
Thanks.

Yes. se try to avoid outputting probabilities before calculating the loss. 

Moved sigmoid to the end, so the print-outs make sense.
